src/Momento.Sdk/Auth/EnvMomentoTokenProvider.cs

```diff
@@ -32,4 +36,11 @@ public EnvMomentoTokenProvider(string name)
         ControlEndpoint = claims.ControlEndpoint;
         CacheEndpoint = claims.CacheEndpoint;
     }
+
+    public ICredentialProvider WithCacheEndpoint(string cacheEndpoint)
```

Comment:
missing documentation here

Reply:
good catch, will fix before moving this out of draft

Reply:
fixed in https://github.com/momentohq/client-sdk-dotnet/pull/348/commits/56c8d2b7de59913bae02b97897b210049e2f5bda